gateway/gateway/Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
-name = "ln-gateway"
+name = "gateway"
 version = "0.1.0"
 authors = ["The Fedimint Developers"]
 edition = "2021"
-description = "ln-gateway is a core lightning plugin which allows a Lightning node operator to receive or pay Lightning invoices on behalf of fedimint users."
+description = "gateway is a core lightning plugin which allows a Lightning node operator to receive or pay Lightning invoices on behalf of fedimint users."
```

Comment:
we should update this description. perhaps this?

_gateway is an extension service that facilitates receiving or paying Lightning invoices on behalf of users in a connected Fedimint federation. Run this if you want to connect a federation to the rest of the bitcoin network, over lightning_

Reply:
let's do that later